btrack/napari/widgets/_general.py

```diff
@@ -1,6 +1,40 @@
 from __future__ import annotations
 
-from qtpy import QtWidgets
+from pathlib import Path
+
+from qtpy import QtCore, QtGui, QtWidgets
+
+
+def create_logo_widgets() -> dict[str, QtWidgets.QWidget]:
+    """Creates the widgets for the title, logo and documentation"""
+
+    title = QtWidgets.QLabel("<h3>Bayesian Tracker</h3>")
+    title.setAlignment(QtCore.Qt.AlignHCenter)
+    widgets = {"title": title}
+
+    logo = QtWidgets.QLabel()
+    logo.setPixmap(
+        QtGui.QPixmap(
+            str(
+                Path(__file__).resolve().parents[3]
+                / "docs"
+                / "_static"
+                / "btrack_logo.png"
```

Comment:
This will likely fail when doing a pypi install, since docs is outside the package. I think we need to add the logo as an asset inside the napari subpackage?